### DISCUSSION TITLE
Where is the static site files?

### DISCUSSION THREAD

**Discussion by jeeftor** [tteck/Proxmox#861] [Help Needed]
I'd like to make a PR to the readme file - but is it in a different repo or am I completely blind?

**Comment by tteck** [comment]
The site (https://tteck.github.io/Proxmox/) is not created from a markdown file.

**Reply by jeeftor** [reply]
> The site (https://tteck.github.io/Proxmox/) is not created from a markdown file.
> 
> 

It's a jekyl task no?
